lib/ansible/modules/remote_management/hpilo/hpilo_boot.py

```diff
@@ -65,12 +65,6 @@
     - As a safeguard, without force, hpilo_boot will refuse to reboot a server that is already running.
     default: no
     choices: [ "yes", "no" ]
-  ssl_version:
-    description:
-      - Change the ssl_version used.
-    default: TLSv1
-    choices: [ "SSLv3", "SSLv23", "TLSv1", "TLSv1_1", "TLSv1_2" ]
-    version_added: '2.4'
```

Comment:
Also, this was added for a specific reason which made the module fail for some users, so removing it will also make the module non-functional for those users. Another reason why this is a NO-GO.

Reply:
@dagwieers  I can implement a conditional clause to use this parameter or not depending of the underlying version of python-hpilo. What do you think?

Reply:
The functionality should still exist with the new python-hpilo, so simply allowing to disable it removes the functionality for people with older iLO systems and a newer python-hpilo. I don't think that's a good proposition.